run0/src/container/command.rs

```diff
@@ -0,0 +1,42 @@
+use oci_spec::runtime::Process;
+
+const UNSHARE_PROGRAM_ENTRYPOINT: &str = "/bin/sh";
+const DEFAULT_ARG0: &str = "sh";
+
+/// Implementation of the container arguments.
+#[derive(Default)]
+pub struct Command {
+    arg0: String,
+    args: Vec<String>,
+}
+
+impl Command {
+    /// Get arguments.
+    /// It converts the `Self` into a `Command` for `unshare` API compatibility.
+    pub fn get(&self) -> unshare::Command {
```

Comment:
What you're trying to do here is to convert a `Command` into an `unshare::Command`, so you should implement the `From` trait instead:

```rust
impl From<&Command> for unshare::Command {
}
```